src/alert/src/Alert.js

```diff
@@ -60,7 +68,8 @@ const Alert = memo(
               marginBottom={0}
               fontWeight={500}
               lineHeight={1}
-              color={intents[intentToken].text}
+              // Get this from the theme / props on the Alert
+              color="inherit"
```

Comment:
Couldn't think of a simpler way to handle this... I dont like reaching into the theme directly here, so I chose to inherit whatever color is defined on the Alert wrapper itself.